### PR TITLE
ECSW Hyper-reduction

### PR DESCRIPTION
**Pull request by pjb236**
Baseline ECSW implementation following Farhat group's work with some tests.

### CODE REVIEW

**Comment by fnrizzi** [comment]
@eparish1 will you take a look?

File: romtools/hyper_reduction/__init__.py [hunk 1]
```diff
@@ -1,6 +1,11 @@
 '''
+<<<<<<< HEAD
```
Comment (fnrizzi) [inline romtools/hyper_reduction/__init__.py:2]: leftover thing from merging :)

**Comment by pjb236** [comment]
@fnrizzi @eparish1 @jtencer Circling back to this branch, any reason I shouldn't merge this into develop?

**Comment by fnrizzi** [comment]
> @fnrizzi @eparish1 @jtencer Circling back to this branch, any reason I shouldn't merge this into develop?

did you fix the leftover git thing in my comment ?

**Comment by fnrizzi** [comment]
i know Eric said he wnated to look at this

**Comment by eparish1** [comment]
@pjb236 I'm just way behind. I haven't had a chance to look over it carefully, but I'm good to merge it.

**Comment by pjb236** [comment]
@eparish1 @fnrizzi Aren't we supposed to merge into develop, not main?

**Comment by pjb236** [comment]
Looks like this was merged into main, not develop. I'm guessing the automated pull request button will merge to main...

**Comment by fnrizzi** [comment]
Yes it was targeting the wrong branch I can fix it

**Pull request merged**